source/_integrations/fritz.markdown

```diff
@@ -142,3 +146,13 @@ automation:
           message: "Password: ..."
 
 ```
+
+## Troubleshooting
+
+In any case, when reporting an issue, please enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics), restart the integration, and as soon as the issue re-occurs stop the debug logging again (_download of debug log file will start automatically_). Further _if still possible_, please also download the {% term diagnostics %} data. If you have collected the debug log and the diagnostics data, provide them with the issue report.
+
+### Device presence detection is not working as expected
+
+You see a device as still present, even it is offline or disconnected for more than the configured [consider home](#consider-home) seconds? You're using additional network equipment like a network switch or wifi access point other than an AVM Fritz!Repeater or other AVM components, but not configured as a [mesh](https://en.avm.de/service/knowledge-base/dok/FRITZ-Box-7590/3329_Mesh-with-FRITZ/) in your home network?
+
+Than you should [enable the old discovery method](#enable-old-discovery-method) in the [integration options](#integration-options).
```

Comment:
```suggestion
If one of the above cases applies to your setup, try [enabling the old discovery method](#enable-old-discovery-method) in the [integration options](#integration-options). This might resolve the issue.
```